charts/seed-controlplane/charts/kube-apiserver-sni/templates/kube-apiserver-envoyfilter.yaml

```
@@ -35,3 +35,31 @@ spec:
           prefix_ranges:
           - address_prefix: {{ .Values.apiserverClusterIP }}
             prefix_len: 32
+  - applyTo: NETWORK_FILTER
+    match:
+      context: GATEWAY
+      listener:
+        name: 0.0.0.0_8132
+        portNumber: 8132
```

This can be changed to `9443` as it would allow `443` port (on the LoadBalancer) to be used.

```suggestion
        name: 0.0.0.0_9443
        portNumber: 9443
```